packages/analytics-browser/test/browser-client.test.ts

```diff
@@ -952,6 +973,33 @@ describe('browser-client', () => {
   });
 
   describe('process', () => {
+    test('should process create new session for new campain', async () => {
+      await client.init(apiKey, {
+        optOut: true,
+        logLevel: LogLevel.Warn,
+        defaultTracking: {
+          attribution: {
+            resetSessionOnNewCampaign: true,
+          },
+        },
+      }).promise;
+
+      expect(client.webAttribution).toBeDefined();
+
+      // Making sure client.webAttribution is not undefined
+      if (!client.webAttribution) {
```

Comment:
Shouldn't this be impossible based on the expect().toBeDefined() above?

Reply:
Yea, this bugs me a lot. the client.webAttribution is defined, but since client.webAttribution can be undefined, it won't let me spy on client.webAttribution.